resources/views/settings/access-needs.blade.php

```diff
@@ -122,7 +124,8 @@
             <legend>
                 <h2>{{ __('For in-person meetings') }}</h2>
             </legend>
-            <p>{{ __('All organizations have been asked to provide gender-neutral and accessible washrooms by default.') }}
+            <p>{{ __('Access needs for when you are attending a meeting in-person.') }}
+                <b>{{ __('Please note that all organizations have been asked to provide gender neutral and accessible washrooms by default.') }}</b>
```

Comment:
It's probably better to use `<strong>` instead of `<b>`. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

```suggestion
                <strong>{{ __('Please note that all organizations have been asked to provide gender neutral and accessible washrooms by default.') }}</strong>
```